unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Scene/SceneController.cs

```diff
@@ -725,9 +700,17 @@ public void UnloadParcelSceneExecute(string sceneKey)
             ProfilingEvents.OnMessageProcessEnds?.Invoke(MessagingTypes.SCENE_DESTROY);
         }
 
-        public void UnloadAllScenes()
+        public void UnloadAllScenes(bool includePersistent = false)
         {
-            var list = Environment.i.worldState.loadedScenes.ToArray();
+            if (includePersistent)
+            {
+                foreach (var keyValuePair in Environment.i.world.state.loadedScenes.Where(x => x.Value.isPersistent))
```

Comment:
In this case, wouldn't it better to store the result of `Environment.i.world.state.loadedScenes.Where(x => x.Value.isPersistent)` in a variable before the loop?

Reply:
Indeed. I just moved the existing code, I agree it's too lazy. The request seems to be harmless so I'm going to try it!

Reply:
Done